core/templates/dev/head/pages/library/Library.js

```diff
@@ -55,7 +58,11 @@ oppia.controller('Library', [
 
     $scope.activeGroupIndex = null;
 
-    $scope.pageMode = GLOBALS.PAGE_MODE;
+    var currentPath = $window.location.pathname;
+    $scope.pageMode = LIBRARY_PATHS_AND_MODES[currentPath];
+    if (!$scope.pageMode) {
```

Comment:
Don't rely on "undefined". Use LIBRARY_PATHS_AND_MODES.hasOwnProperty(currentPath) instead to check.

Reply:
Done.